wpengine/faustjs
# ✨ New Example: Block Support

## Tasks


## Description

These changes introduce a new example project that demonstrates how to configure block support with the Faust.js framework.

## Testing

1. Checkout this branch: `nojira-block-support-example`
2. Run `npm run dev -w @faustwp/block-support-example`
3. Observe that block-based posts/pages are properly rendering
